NOnion/Cells/CellCerts.fs

```diff
@@ -44,26 +42,22 @@ type CellCerts =
 
     interface ICell with
 
-        member __.Command = 129uy
+        member self.Command = 129uy
```

Comment:
@symboliq why are you removing the double underscores?

Reply:
Accidental merge issue. Will fix.

Reply:
I'll also push back on this convention with a PR. `__` is just as much a binding as `self` is - and is also equally in the VS variable inspector. We need to use `this` because it allows us to do normal variable inspection while debugging.